src/cluster/setup.rst

```diff
@@ -167,6 +167,10 @@ Before you can add nodes to form a cluster, you have to have them
 listen on a public IP address and set up an admin user. Do this, once
 per node:
 
+Change line ``-name couchdb@127.0.0.1`` in file /Couch-Home/etc/vm.args on
+each node for clustered setup, each node in system must have a unique name.
+eg. ``-name couchdb@couch1`` | ``-name couchdb@couch2``
```

Comment:
The `couch1`, `couch2` hostnames must be fully qualified domain names (FQDNs) with valid DNS entries, see [the comment in vm.args](https://github.com/apache/couchdb/blob/master/rel/overlay/etc/vm.args#L13-L28) for more detail.